### ISSUE TITLE
about SPN network in paper

### ISSUE THREAD

**Issue by zhui064** [JingchunCheng/Seg-with-SPN#1]
The paper proposes the spatial propagation network (SPN)  as a refinement module, which learns the affinity entities and refine the coarse mask. But I didn't find the SPN code and pretrained SPN model. Could you submit the corresponding code for us.

**Comment by wasidennis** [comment]
Thanks for your interest. Currently we are working on the IP issue for SPN, and will upload the pre-trained model once the issue is solved.
